Tests/PointFreeTests/__Snapshots__/AuthTests/testLogin.1.Conn.txt

```diff
@@ -9,4 +9,4 @@
 
 ▿ Response
   Status 302 FOUND
-  Location: https://github.com/login/oauth/authorize?scope=user:email&redirect_uri=http://localhost:8080/github-auth%3Fcode%3D&client_id=deadbeef-client-id
\ No newline at end of file
+  Location: https://github.com/login/oauth/authorize?scope=user:email&client_id=deadbeef-client-id&redirect_uri=http://localhost:8080/github-auth
```

Comment:
I forget, are we supposed to sort params?

Reply:
ugh well, if it's just plain key/value pairs that are strings, then that's the best way to ensure well-defined rendering. but for more complicating query strings, like `a[]=1&a[]=2` we can't just sort.

so it's complicated...